core/src/main/res/values-fr/strings.xml

```diff
@@ -126,13 +126,13 @@
     <string name="chart_filter_type_hint">Type de statistiques</string>
     <string name="chart_filter_type_activity">Activité</string>
     <string name="chart_filter_type_category">Étiquette</string>
-    <string name="chart_filter_categories_empty">Aucune Étiquettes ajoutée</string>
+    <string name="chart_filter_categories_empty">Aucune étiquette ajoutée</string>
 
     <string name="duration_dialog_save">Sauvegarder</string>
     <string name="duration_dialog_disable">Désactiver</string>
 
-    <string name="color_selection_base_color_hint">Sélectionnez la couleur de base</string>
-    <string name="color_selection_adjust_color_hint">Ajuster la couleur</string>
+    <string name="color_selection_base_color_hint">Sélectionnez la teinte de base</string>
+    <string name="color_selection_adjust_color_hint">Ajustez la couleur</string>
```

Comment:
Not sure about this, in english "base color" maybe sounds better because it is simpler and more understandable than "hue" which sounds like a specific term. But I'm not a native english speaker nor french speaker so I'll trust you with this :)

Reply:
On second thought, I think it's better to keep *couleur de base* (base color) which is really simpler and therefore clearer! On a related note, according to Google Ngrams, in English [`color` is around 20 times more common than `hue`](https://books.google.com/ngrams/graph?content=color%2Chue&year_start=2000&year_end=2019&case_insensitive=on&corpus=en-2019&smoothing=3), but in French [*couleur* is only 10 times more common than *teinte*](https://books.google.com/ngrams/graph?content=couleur%2Cteinte&year_start=2000&year_end=2019&case_insensitive=on&corpus=fr-2019&smoothing=3).